docs/docs.md

```diff
@@ -0,0 +1,3 @@
+---
+title: Docs
+---
```

Comment:
Not sure what content we should put here. Most basic would be a list reflecting the links in the sidebar.

Reply:
Maybe? Do we need to put something here?